pkg/imageset/updateinstaller.go

```diff
@@ -174,6 +174,12 @@ func (o *UpdateInstallerImageOptions) Run() (returnErr error) {
 		if cd.Spec.Platform.BareMetal != nil {
 			installerTagName = "baremetal-installer"
 		}
+		// Override annotation is allowed to override baremetal-installer too
+		if cd.Annotations != nil {
+			if override := cd.Annotations[constants.OverrideInstallerImageNameAnnotation]; override != "" {
+				installerTagName = override
+			}
```

Comment:
```suggestion
			if override, found := cd.Annotations[constants.OverrideInstallerImageNameAnnotation]; found {
				installerTagName = override
			}
```
perhaps more idiomatic

Reply:
In this case I'm treating "not provided" and "explicit empty string" the same: I'll fall back to the default. With your suggestion, in the latter case I would actually attempt to look up an image named `""` which would presumably fail. Are you saying that would be the preferred behavior? As a kind of stopgap for a buggy consumer accidentally providing an "incorrect" (in this case, empty) image name to the annotation?

Reply:
>  Are you saying that would be the preferred behavior?

Nope. I agree with your conclusion. 👍